docs/docs/en/development/development-environment-setup.md

```diff
@@ -5,10 +5,9 @@
 Before setting up the DolphinScheduler development environment, please make sure you have installed the software as below:
 
 * [Git](https://git-scm.com/downloads): DolphinScheduler version control system
-* [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html): DolphinScheduler backend language
+* [JDK1.8](https://www.oracle.com/technetwork/java/javase/downloads/index.html): DolphinScheduler backend language
```

Comment:
I prefer using content in the tail of this line, because the link https://www.oracle.com/technetwork/java/javase/downloads/index.html could download all version of jdk, and is not specific to version 1.8. same as `Node`
```suggestion
* [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html): DolphinScheduler backend language, have to use version after JDK 1.8
```

Reply:
i wlii fix it.